TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auton/colorSensor/colorSensorTest.java

```diff
@@ -0,0 +1,23 @@
+package org.firstinspires.ftc.teamcode.auton.colorSensor;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.ColorSensor;
+
+public class colorSensorTest extends LinearOpMode {
+    ColorSensor color_sensor;
+
+    public void runOpMode() {
+        color_sensor = hardwareMap.colorSensor.get("color");
```

Comment:
let's make an object called color_sensor and a separate value (eg. color_value) to hold the reading.